README.md

```diff
@@ -15,3 +20,31 @@ Runtime provides Java interfaces like `IJob`, `ISchedulingRule` and implements s
 ### [eclipse.update](https://github.com/eclipse-platform/eclipse.platform/tree/master/update)
 Update provides Java interfaces like `IPlatformConfiguration`. `IPlatformConfiguration` represents the runtime configuration.
 
+
+## How to Contribute
+
+Contributions are most welcome. There are many ways to contribute, from entering high quality bug reports, to contributing code or documentation changes.
+For a complete guide, see the https://github.com/eclipse-platform/.github/blob/main/CONTRIBUTING.md.
+
+
+## Issue Tracking
```

Comment:
i would prefer a link to https://github.com/eclipse-platform#reporting-issues

Reply:
Yes, the link should definitely be there. So would you like to have that link in addition to the three links for finding and creating new issues, or as replacement for the link to submit a new issue to this repository?

Reply:
don't know. somehow avoid duplication. the link to the parent o https://github.com/eclipse-platform is already included in this Readme. Maybe just make it more prominent that that link is relevant for those topics you seem to be missing?

Reply:
I have created https://github.com/eclipse-platform/.github/pull/121 to improve the central organization's readme with a link to finding issues across all organization repositories (which I had formerly added here) to avoid duplication of that information across repositories.

I have updated the proposed change to have a reference to the central section about reporting issues in the organization's readme thatyou mentioned, @jukzi, in the first place.
The links to finding and creating issues within this repository are mentioned afterwards in case someone knows the issue fits into this repository.

This avoids redundancy w.r.t. information already given in the organization's readme. Having a link to both the general organization page and the specific readme section about reporting issues does not seem to be a problem of duplication to me, but eases finding appropriate information in the according sections?

Reply:
I am missing a statement about the community here as in the parent. I don't like to repeat it here, but i would like potential reporters to read it first to avoid issues created without that in mind.

Reply:
Do you refer to the information in https://github.com/eclipse-platform#community, so the section above the one about reporting issues? Would make totally sense to say that you have to keep notice of the information given there before submitting an issue.

What about something like:
> This project uses Github to track ongoing development and issues. In case you have an issue, please read the information about Eclipse being a [community project](https://github.com/eclipse-platform#community) and bear in mind that this project is almost entirely developed by volunteers. So the contributors may not be able to look into every reported issue. You will also find the information about [how to find and report issues](https://github.com/eclipse-platform#reporting-issues) in repositories of the `eclipse-platform` organization there.
In case you want to report an issue that is specific to this `eclipse.platform` repository, you can [find existing issues](https://github.com/eclipse-platform/eclipse.platform/issues) or [create new issues](https://github.com/eclipse-platform/eclipse.platform/issues/new) within this repository.

That should not be too repetitive, but summarizes the important information and links to the more verbose source of information.

Reply:
ok

Reply:
Thanks! I have update the proposed change.